apps/gruenbuch/models.py

```diff
@@ -10,17 +10,23 @@
 from apps.contrib.translations import TranslatedField
 
 class GruenbuchIndexPage(Page):
-    template = 'gruenbuch_index_page.html'
+    template = 'gruenbuch_page.html'
 
     def get_context(self, request):
         context = super().get_context(request)
-        context['gruenbuch_sections'] = GruenbuchDetailPage.objects.live().public()
+        context['book'] = self
+        context['chapters'] = self.get_children().specific()
+        context['active_chapter'] = context['chapters'].first()
+        context['next_chapter'] = context['active_chapter'].get_next_sibling()
+        if self.get_next_sibling():
+            context['next_chapter'] = self.get_next_sibling().specific
+
```

Comment:
Instead of adding stuff to the context, you can also use properties, which can then be used like the fields of the page. Maybe that's nicer here.
```
@property
def chapters(self):
    return self.get_children().specific()
```